ecl/baremetal/v2/_proxy.py

```diff
@@ -364,15 +364,19 @@ def start_server(self, server_id, boot_mode):
         server = _server.ServerAction()
         return server.start(self.session, server_id, boot_mode)
 
-    def stop_server(self, server_id):
+    def stop_server(self, server_id, type):
```

Comment:
type is optional so should use default value.
```suggestion
    def stop_server(self, server_id, type=None):
```